packages/hydrogen-netlify-plugin/index.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require('node:fs')
+
+function onBuild() {
+  fs.mkdirSync('./.netlify/edge-functions/', { recursive: true })
+  fs.copyFileSync('./dist/worker/index.js', './.netlify/edge-functions/server.js')
+
+  fs.mkdirSync('./public/build/', { recursive: true })
+  fs.cpSync('./dist/client/build/', './public/build/', { recursive: true })
+
+  // eslint-disable-next-line no-console
+  console.log(
+    `Copying built Hydrogen/Remix project files for Netlify Edge Function bundling (Current directory: '${process.cwd()}').`,
+  )
```

Comment:
Rather than copying the build ooutput, are we able to change the remix config to emit them in the right place?

Reply:
I don't believe we can because Hydrogen has their own build process outside of the Remix boundary. Correct me if I'm wrong about that @zachleat.

Reply:
Aah, how did I miss that!